packages/lisk-transactions/src/0_transfer.ts

```diff
@@ -44,81 +30,48 @@ export interface TransferInputs {
 	readonly timeOffset?: number;
 }
 
-const validateInputs = ({
```

Comment:
we should keep this `validateInputs` to this file as it's only used here

Reply:
Done. Resolving.